polygon-digital-carbon/.gitignore

```diff
@@ -0,0 +1,73 @@
+# Logs
+logs
```

Comment:
it's unusual to have multiple `.gitignore` files in a single repo... not going to make a big deal about it but we should really only have one at the top-level of the repo